lardataobj/RawData/raw.h

```diff
@@ -90,6 +92,17 @@ namespace raw{
   void UncompressHuffman(const std::vector<short>& adc,
                          std::vector<short>      &uncompressed);
 
+  short fibonacci_decode(std::vector<bool>& chunk);
+  void fibonacci_encode_table(int end, std::vector<std::vector<bool>>& table);
```

Comment:
Two questions:
- Why use `std::vector<bool>`?  Such a type is highly specialized and often discouraged.
- Can the function be changed to return the container instead of passing it in by non-`const` reference?

Reply:
Well, `fibonacci_encode_table` is meant to update the table. 
For the `std::vector<bool>` I don't know, to be honest. If you have a better option I'll take it.

Reply:
To be honest, I'm not sure I have a better suggestion for either issue.  Let's leave it alone for now.